src/lumigo_tracer/user_utils.py

```diff
@@ -11,3 +13,38 @@ def report_error(msg: str):
     else:
         message_with_request_id = f"RequestId: {SpansContainer.get_span().function_span.get('id')} {message_with_initials}"
         print(message_with_request_id)
+
+
+def add_execution_tag(key: str, value: str, should_log_errors: bool = True) -> None:
+    """
+    The maximum number of tags is 50.
+    :param key: Length should be between 1 and 50.
+    :param value: Length should be between 1 and 50.
+    :param should_log_errors: Should a log message be printed in case the tag can't be added.
+    """
+    try:
+        tags_len = SpansContainer.get_span().get_tags_len()
+        key = str(key)
+        value = str(value)
+        if not key or len(key) >= MAX_TAG_KEY_LEN:
+            if should_log_errors:
+                warn_client(
+                    f"Unable to add tag: key length should be between 1 and {MAX_TAG_KEY_LEN}: {key} - {value}"
```

Comment:
add `Skipping` or something like that, so the client will know that we're not adding this key, but everything  should continue as expected